charms/worker/k8s/charmcraft.yaml

```diff
@@ -161,6 +161,32 @@ config:
         CIDR to use for Kubernetes services. After deployment it is
         only possible to increase the size of the IP range. It is not possible to
         change or shrink the address range after deployment.
+    local_storage_enabled:
+      type: boolean
+      description: |
+        Enable local storage provisioning. This will create a storage class
+        named "local-storage" that uses the hostPath provisioner. This is
+        useful for development and testing purposes. It is not recommended for
+        production use.
+    local_storage_local_path:
+      type: string
+      default: ""
```

Comment:
let's go ahead and copy through the default value from the snap shall we?  is that ""?  What path does this end up consuming?